packages/financial-templates-lib/test/clients/FinancialContractEventClient.js

```diff
@@ -329,22 +360,22 @@ contract("FinancialContractEventClient.js", function (accounts) {
               objectsInArrayInclude(
                 [
                   {
-                    transactionHash: newSponsorTxObj1.tx,
-                    blockNumber: newSponsorTxObj1.receipt.blockNumber,
```

Comment:
pretty irritating that they change this syntax. with that said, the hardhat syntax is cleaner.

Reply:
Yeah, agreed. Was pretty annoying to go and fix all of these.